pkg/acsengine/const.go

```diff
@@ -87,7 +87,7 @@ var KubeImages = map[string]map[string]string{
 		"dashboard":       "kubernetes-dashboard-amd64:v1.6.1",
 		"exechealthz":     "exechealthz-amd64:1.2",
 		"addonresizer":    "addon-resizer:1.7",
-		"heapster":        "heapster:v1.4.0",
+		"heapster":        "heapster-arm64:v1.4.0",
```

Comment:
Just to be aligned with other images. Also kubernetes started using arm64. https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/cluster-monitoring/google/heapster-controller.yaml#L48